client/src/Ratings.jsx

```diff
@@ -1,59 +1,65 @@
+/* eslint-disable import/extensions */
 import React, { Component } from 'react';
-import PropTypes from 'prop-types';
+import axios from 'axios';
+
 import ReviewAction from './RatingsComponent/ReviewAction.jsx';
 import RatingsSearch from './RatingsComponent/RatingsSearch.jsx';
 import RatingsContent from './RatingsComponent/RatingsContent.jsx';
 import RatingsStarHeader from './RatingsComponent/RatingsStarHeader.jsx';
 import RatingsSummaryReview from './RatingsComponent/RatingsSummaryReview.jsx';
-const { data } = require('./RatingsComponent/RatingsDummyData.js');
-
-let newData = data.slice(0,2);
 
 class Ratings extends Component {
-  constructor(props) {
-    super(props);
+  constructor() {
+    super();
     this.state = {
-      ratings: newData,
+      ratingsStorage: [],
+      limitedRatings: [],
```

Comment:
What is the point of ratings storage amd limited Ratings both being in state? Is there a reason to keep the two variables.